docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -27,20 +27,20 @@ Object.entries(frontMatter).filter(
 
 # Getting started with Semgrep App
 
-Semgrep App supports code scanning from:
-
-* local command-line interfaces (CLI)
-* source code management (SCM) systems, such as GitHub and GitLab, through continuous integration (CI)
-
-This guide walks you through scanning code in both types of environments.
+Semgrep App enables you to run scans on multiple repositories by integrating with your GitHub or GitLab SaaS account. Semgrep uses **rules** to scan code. Matches found based on those rules are called **findings**. A Semgrep rule encapsulates pattern-matching logic and data-flow analysis used to find code violations, security issues, outdated libraries, and other issues.

```

Comment:
Non-blocking and optional - just to avoid "and other issues" (similar to etc - https://developers.google.com/style/word-list#etc)
```suggestion
Semgrep App enables you to run scans on multiple repositories by integrating with your GitHub or GitLab SaaS account. Semgrep uses **rules** to scan code. Matches found based on those rules are called **findings**. A Semgrep rule encapsulates pattern-matching logic and data-flow analysis used to find vulnerabilities such as code violations, security issues, or outdated libraries.
```